packages/amplify_authenticator/lib/src/l10n/input_resolver.dart

```diff
@@ -423,30 +433,37 @@ class InputResolver extends Resolver<InputResolverKey> {
     }
   }
 
+  /// Returns the [String] displayed as the input hint.
   String hint(BuildContext context, InputField field) {
     final fieldName = title(context, field);
     final lowercasedFieldName = fieldName.toLowerCase();
     return AuthenticatorLocalizations.inputsOf(context)
         .promptFill(lowercasedFieldName);
   }
 
+  /// Returns the [String] used as the hint for the password confirmation field.
```

Comment:
This is a general confirmation text, not specific to password

Reply:
It could be used generally, true, but right now the fieldName is being set to `AuthenticatorLocalizations.inputsOf(context).password`, and the only place it's used is password confirmation:
```dart
  static const passwordConfirmationHint = InputResolverKey._(
    InputResolverKeyType.confirmHint,
    field: InputField.passwordConfirmation,
  );
  ```

Reply:
Right, but the function signature takes in an `InputField`. When you say password confirmation here, it's confusing how to implement the method for a generic field. This method just produces `Re-enter ${field}` so any overrides should do the same.

Reply:
You can probably make the doc a little more generic without losing the info about it being used for password confirmation. Something like
```suggestion
  /// Returns the [String] used as the hint for a confirmation field, for example, a password confirmation field.
```

Reply:
Ok, fair enough.